eventarc/storage-handler/README.md

```diff
@@ -0,0 +1,89 @@
+# Eventarc - Cloud Storage Events
+
+This sample shows how to create a service that processes GCS events.
+
+For more details on how to work with this sample read the [Google Cloud Run Java Samples README](https://github.com/GoogleCloudPlatform/java-docs-samples/tree/main/run).
+
+[![Run in Google Cloud][run_img]][run_link]
+
+## Dependencies
+
+* **Spring Boot**: Web server framework.
+* **Jib**: Container build tool.
+* **Junit + SpringBootTest**: [development] Test running framework.
+* **MockMVC**: [development] Integration testing support framework.
+
+## Setup
+
+Configure environment variables:
+
+```sh
+export MY_RUN_SERVICE=gcs-service
+export MY_RUN_CONTAINER=gcs-container
+export MY_GCS_TRIGGER=gcs-trigger
+export MY_GCS_BUCKET="$(gcloud config get-value project)-gcs-bucket"
+export SERVICE_ACCOUNT=gcs-trigger-svcacct
+export PROJECT_ID=$(gcloud config get-value project)
+```
+
+## Quickstart
+
+Use the [Jib Maven Plugin](https://github.com/GoogleContainerTools/jib/tree/master/jib-maven-plugin) to build and push your container image:
+
+```sh
+mvn jib:build -Dimage=gcr.io/$PROJECT_ID/$MY_RUN_CONTAINER
+```
+
+Deploy your Cloud Run service:
+
+```sh
+gcloud run deploy $MY_RUN_SERVICE \
+--image gcr.io/$PROJECT_ID/$MY_RUN_CONTAINER \
+--region us-central1
```

Comment:
I think this may require the unauthenticated flag, but do we want to update the trigger so we can keep this requiring auth?

Reply:
If we give the service account `roles/run.invoker`, we can leave auth enabled on the run service. i've updated the readme to add that role.